packages/calypso-e2e/test/lib/utils/get-account-by-feature.test.ts

```diff
@@ -89,8 +89,14 @@ describe( 'getTestAccountByFeature', function () {
 	} );
 
 	it( 'will throw en error if passed feature does not match an account', () => {
+		// Trying to maintain this test over time will be tedious if we stick to allowed values.
+		// We might not support a combination now, but may in the future, which would cause this test to fail.
+		// So, we use a bad value and override the typing to check the error throwing.
 		expect( () =>
-			getTestAccountByFeature( { gutenberg: 'edge', siteType: 'atomic' } )
+			getTestAccountByFeature( ( {
+				gutenberg: 'bad_value',
+				siteType: 'atomic',
+			} as unknown ) as FeatureCriteria )
```

Comment:
The jest config was messed up, so this suite wasn't actually being run! This test was failing, so I fixed it.